### PR TITLE
Fix attr getters in Network Resource

### LINKED ISSUE #527: [BUG] Incorrect key used in the Network.containers property
The dictionary query `self.attrs["Containers"]` at [networks.py#L42-L48](https://github.com/containers/podman-py/blob/main/podman/domain/networks.py#L42-L48)  seems to be incorrect (also shown below). Using the key `"containers"` (lower-case "c") fixes the issue. 

```python
    @property
    def containers(self):
        """list[Container]: Returns list of Containers connected to network."""
        with suppress(KeyError):
            container_manager = ContainersManager(client=self.client)
            return [container_manager.get(ident) for ident in self.attrs["Containers"].keys()]
        return []
```

If fixing this issue is truly as simple as changing the key used to be `"containers"` instead, let me know and I can make a PR (I can also spend some time to look into adding some unit tests for this).

### PR DESCRIPTION
**Pull request by nlacasse**
The `id` and `containers` attrs should be lowercase.

Fixes #527

### CODE REVIEW

**Comment by nlacasse** [comment]
Sorry for not adding tests, but for some reason none of the tests run for me.

They all loop forever with messages like:
```
2025-08-27 21:08:15 [    INFO] Launching(4.9.3) podman --log-level=info system service --time=0 unix:///tmp/tmpocgy0b6d/498a901d0d504c4e9b302f93e369569c refid=1290867612650871671 (utils.py:86)
2025-08-27 21:08:15 [   DEBUG] Starting new HTTP connection (1): localhost:22 (connectionpool.py:241)
2025-08-27 21:08:15 [   DEBUG] Waiting on /run/user/1000/podman/podman-forward-7e5394932fe630968a58.sock (ssh.py:89)
2025-08-27 21:08:15 [   DEBUG] Waiting on /run/user/1000/podman/podman-forward-7e5394932fe630968a58.sock (ssh.py:89)
2025-08-27 21:08:16 [   DEBUG] Waiting on /run/user/1000/podman/podman-forward-7e5394932fe630968a58.sock (ssh.py:89)
2025-08-27 21:08:16 [   DEBUG] Waiting on /run/user/1000/podman/podman-forward-7e5394932fe630968a58.sock (ssh.py:89)
2025-08-27 21:08:16 [   DEBUG] Waiting on /run/user/1000/podman/podman-forward-7e5394932fe630968a58.sock (ssh.py:89)
2025-08-27 21:08:16 [   DEBUG] Waiting on /run/user/1000/podman/podman-forward-7e5394932fe630968a58.sock (ssh.py:89)
2025-08-27 21:08:16 [   DEBUG] Waiting on /run/user/1000/podman/podman-forward-7e5394932fe630968a58.sock (ssh.py:89)
2025-08-27 21:08:17 [   DEBUG] Waiting on /run/user/1000/podman/podman-forward-7e5394932fe630968a58.sock (ssh.py:89)
...
```

I followed the steps in `CONTRIBUTING`, so I'm not sure what could be wrong.

**Comment by inknos** [comment]
if you are able to `ssh localhost` the error should go away. we should correct the `CONTRIBUTING`

File: podman/domain/networks.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class Network(PodmanResource):
     def id(self):  # pylint: disable=invalid-name
         """str: Returns the identifier of the network."""
         with suppress(KeyError):
-            return self.attrs["Id"]
+            return self.attrs["id"]
```
Comment (inknos) [inline podman/domain/networks.py]: this line is causing many troubles in the tests, I suggest reverting it

**Review by inknos — CHANGES_REQUESTED** [review]
@nlacasse thanks for the PR

A couple of points

- what is the reason of returning "id" instead of "Id"?
- can we just change "containers" in this PR? let's focus on fixing the tests and have them running consistently on your machine and in this PR, what do you say? :)

File: podman/domain/networks.py (hunk 1 above)
Comment (nlacasse) [reply podman/domain/networks.py]: Done.

**Comment by nlacasse** [comment]
Thanks!  I reverted the `Id` -> `id` change.  I think something weird is still going on there, but I'll investigate further and open a separate PR if needed.

Also the tests are working fine for me now.  Not sure what was going on before.  Sorry for the noise.

**Comment by nlacasse** [comment]
I filed a [new bug](https://github.com/containers/podman-py/issues/584) for the `Id` issue.

**Comment by inknos** [comment]
> Also the tests are working fine for me now. Not sure what was going on before. 

tests were off because of Id, mostly. I didn't investigate through all of them though

> Sorry for the noise.

no problem

---

@nlacasse checks are green, that's awesome! DCO check is still complaining, but you commit looks well forged and signed. maybe, try to rebase to main and push again?

**Comment by nlacasse** [comment]
> @nlacasse checks are green, that's awesome! DCO check is still complaining, but you commit looks well forged and signed. maybe, try to rebase to main and push again?

Done.  DCO seems happy now.
